src/server/constants.ts

```diff
@@ -23,17 +24,24 @@ if (!PG_CONNECTION) {
   pgConn.password = PG_META_DB_PASSWORD
   pgConn.pathname = encodeURIComponent(PG_META_DB_NAME)
   pgConn.searchParams.set('sslmode', PG_META_DB_SSL_MODE)
-  if (PG_META_DB_SSL_ROOT_CERT_PATH) {
-    pgConn.searchParams.set('sslrootcert', PG_META_DB_SSL_ROOT_CERT_PATH)
-  }
   PG_CONNECTION = `${pgConn}`
 }
 
+export const PG_META_DB_SSL_ROOT_CERT = process.env.PG_META_DB_SSL_ROOT_CERT
```

Comment:
Where is this getting picked up and used for the connections?

Reply:
I keep getting brain farts... should be set now in DEFAULT_POOL_CONFIG